README.md

```diff
@@ -59,15 +59,15 @@ All packages are available through the pip repository and can be installed via
 <!-- TODO: Explain how this sample can be used and what is required to get it running -->
 # vault.py
 
-This is the low level wrapper for Vault SDK to python. It implements a subset of the available C library as Python classes. Not all functionality
-of Vault SDK is currently implemented in this file. See the Vault SDK documentation for a full list of exposed functions.
+This is the low level wrapper for udSDK to python. It implements a subset of the available C library as Python classes. Not all functionality
+of udSDK is currently implemented in this file. See the Vault SDK documentation for a full list of exposed functions.
 
 #### main.py
 
 This file contains the basic usage of vault SDK for rendering a UDS and writing it to file using Pillow.
-It demonstrates low level usage of functions and is a good starting point for users wanting to fully understand the process of generating a render using Vault SDK
+It demonstrates low level usage of functions and is a good starting point for users wanting to fully understand the process of generating a render using udSDK
 
-Usage: `python3 main.py yourvaultusername yourvaultpassword [pathtoUDS]` will run the sample code and write an image using the Vault Renderer
+Usage: `python3 main.py yourudStreamusername yourudStreampassword [pathtoUDS]` will run the sample code and write an image using the Vault Renderer
```

Comment:
Really should look into using Python to prompt for this information instead of getting it saved on the command line.